src/components/Footer/index.js

```diff
@@ -1,13 +1,41 @@
-import {Box, Button, Grid, Heading} from "grommet";
+import {Box, Button, Grid, Text} from "grommet";
 import React from "react";
 import styled from 'styled-components'
 
 
-export const JoinSlack = () => {
+export const JoinUs = () => {
```

Comment:
Did you do this based on the design? The footer should have been left unchanged.